ang/civicase/CaseDetails--tabs--summary.html

```diff
@@ -44,6 +44,78 @@
     </div>
   </div>
   <!-- End - Subject and QuickLinks -->
+  <div class="civicase__summary-tab-row clearfix">
+    <div class="civicase__summary-tab-tile col-lg-6">
+      <!-- next milestone card -->
+      <h3> Next Milestone </h3>
+      <div class="civicase__activity-card civicase__activity-card--big">
+        <div class="panel panel-default">
+          <div class="panel-body">
+            <!-- ActivityICon directive -->
+            <span class="civicase__activity-icon-container civicase__activity-icon-container--ribbon">
+              <span>
+                <i class="text-danger civicase__activity-icon fa fa-bell"></i>
+              </span>
+              <!-- Activity Ribbon -->
+              <span class="text-danger civicase__activity-icon-ribbon"></span>
+            </span>
+            <!-- ActivityICon directive End -->
+            <!-- Activity Body -->
+            <div class="civicase__activity-card-inner">
+              <!-- Top section -->
+              <div class="civicase__activity-card-row">
+                <span class="civicase__activity-title">Mental evaluation lorem ipsum dolor sit enim</span>
+                <span ng-if="item.activity_summary.milestone[0].assignee_contact_name" contact-card avatar="true" contacts="item.activity_summary.milestone[0].assignee_contact_name"></span>
+                <div ng-include="'~/civicase/ActivityCard--Menu.html'"></div>
+              </div>
+              <!-- End - top section -->
+                <!-- Description -->
+              <div class="civicase__activity-subject">Subject ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam nim ipsam voluptatem quia voluptas sit aspernatur.</div>
+              <!-- End - description -->
+              <!-- Tags -->
+              <div class="civicase__activity__tags-container">
+                <span class="badge" style="background-color: red;">lorem</span>
+                <span class="badge" style="background-color: red;">ipsum</span>
+              </div>
+              <!-- End - Tags -->
+              <!-- With -->
+              <div class="civicase__activity-card-row">
+                <span class="civicase__activity-with">With:&nbsp;</span>
+                <span class="civicase__contact-card" contact-card contacts="item.activity_summary.milestone[0].target_contact_name"></span>
+              </div>
+              <!-- End - With --> 
+              <!-- Big Date -->
+              <div class="civicase__big-date civicase__big-date--overdue">
```

Comment:
As in `ActivityCard.html` we have, `civicase__activity-date`. It should be called `civicase__activity-date--big`.

Reply:
After looking at this deeply I found out that `civicase__activity-date` just uses date as normal inline text with grey color. But this big date component is not a overrided styles or a variant of that date but a different CSS component in itself.

Moving forward when we will move components to the shoreditch it will be easy to consider and move this as a component in shoreditch.
Please let me know if you have deviated thoughts.